README.md

```diff
@@ -73,6 +73,72 @@ Docker is a container system. Click the appropriate link below to install Docker
 
 Note that a full installation of the Docker Toolbox includes Git as well, which is also needed to work on the backend and is mentioned below.
 
+### Virtual Machine (AWS)
```

Comment:
I just created a new [dedicated section](https://github.com/OperationCode/operationcode_backend#setting-up-your-environment) for these AWS Virtual Machine instructions to live under.  You will need to rebase on master to see them.

Please follow the pattern in PR https://github.com/OperationCode/operationcode_backend/pull/277, and relocate these instructions to a new dedicated file called `aws_vm_setup.md`.

Reply:
ok got it